packages/owl-bot-post-processor/package.json

```diff
@@ -4,6 +4,9 @@
   "description": "keep pull requests up-to-date with generated code",
   "bin": "./build/src/bin/owl-bot.js",
   "main": "build/src/index.js",
+  "engines": {
+    "node": ">=10.0.0"
```

Comment:
provide a hint about Node version, so that I can use newish features like `assert.rejects`.